### DISCUSSION TITLE
Great Solution for me!

### DISCUSSION THREAD

**Discussion by gzi01** [softwarecrash/Victron2MQTT#11] [General]
I have four Victron SmartSolar charge controlles and for some reasons they are remote from my battery and inverter. Therefore Victron2MQTT for me is a perfect and affordable solution to get charge controller data into my influx database (raspberry with solaranzeige.de image). Thank you for this great project!

**Comment by softwarecrash** [comment]
thanks you a lot :)
great that it so usefull for you
